### PR TITLE
Use yelp-tools instead of gtk-doc-utils.

### PR DESCRIPTION
**Pull request by pkern**
Apparently GNOME is migrating to yelp-tools away from gtk-doc-utils: https://wiki.gnome.org/Initiatives/GnomeGoals/NewDocumentationInfrastructure

I was pointed at that because of deprecation efforts on the Debian side. This commit implements the move but drops the conditional in configure. I'm not sure if this was required by the Windows build or something else, on the other hand the example didn't have a conditional and I don't know in what depths of m4 I'd need to go to make this work again. But if this is a blocker, please let me know.

### CODE REVIEW

File: help/Makefile.am [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-	      figures/preferences_view.png
-
-DOC_LINGUAS = 
-
+HELP_ID = gobby
+HELP_LINGUAS = 
+HELP_FILES = index.docbook
+HELP_MEDIA = figures/main_window.png      \
+             figures/preferences_editor.png \
+	     figures/preferences_user.png \
```
Comment (aburgm) [inline help/Makefile.am]: some tab/spaces inconsistency here

**Comment by pkern** [comment]
At some point I'll understand why Github just doesn't mail me about things I actually do care about.

I suppose I could skip the help subdir completely in case enable-docs is no. Then the change would be mostly to guard the m4 call just like you did with gnome-doc-utils, that's true. So there's no documentation on Windows at all?

**Comment by aburgm** [comment]
Indeed, there's no documentation on Windows.

**Comment by pkern** [comment]
So like this? The conditional defaults to yes now as auto cannot be supported in the old way.

**Comment by aburgm** [comment]
Looks good!

**Pull request merged**